WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -356,7 +356,7 @@ def getQueues(self, resourceDict):
               self.queueDict[queueName]['ParametersDict'][tagFieldName] = queueTags
             if ceTags:
               if queueTags:
-                allTags = list(set(ceTags + queueTags))
+                allTags = list(set(ceTags | queueTags))
```

Comment:
Is this correct? I'm fairly sure `ceTags` and `queueTags` are always lists so `ceTags.__or__` won't be defined:
```
In [1]: from DIRAC.Core.Utilities.List import fromChar
   ...: ceTags = fromChar("/cvmfs/lhcb.cern.ch,/cvmfs/lhcbdev.cern.ch")
   ...: queueTags = fromChar("aaa,bbb")
   ...: list(set(ceTags | queueTags))
   ...:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-b6cd266fd51c> in <module>()
      2 ceTags = fromChar("/cvmfs/lhcb.cern.ch,/cvmfs/lhcbdev.cern.ch")
      3 queueTags = fromChar("aaa,bbb")
----> 4 list(set(ceTags | queueTags))

TypeError: unsupported operand type(s) for |: 'list' and 'list'
```

Reply:
You are right, the bug was actually introduced in v7r1 somewhere, I will redo the PR.